### PR TITLE
add documentation on a11y for Icon component

### LINKED ISSUE #1567: Add accessibility section for `icon`
### Subject

accessibility section for `icon`

#### Create an accessibility section referring to the links below
https://yamada-ui.com/components/disclosure/tabs/accessibility#aria-roles-and-attributes ( https://github.com/yamada-ui/yamada-ui/issues/1552 )

#### Include the following content
* aria, tabindex property is supported

### Are you willing to participate in creating this documentation page and create a pull request?

None

### PR DESCRIPTION
**Pull request by kurakke**
Closes #1567 

## Description
Add accessibility section for `icon`

## Is this a breaking change (Yes/No):
No

### CODE REVIEW

File: docs/contents/components/media-and-icons/icon/accessibility.ja.mdx [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+with_description: true
+tab: アクセシビリティ
+---
+
+## ARIAロールと属性
+
+| コンポーネント | ロールと属性         | 使い方                                                                            |
+| -------------- | -------------------- | --------------------------------------------------------------------------------- |
+| `Icon`         | `aria-hidden="true"` | 要素をアクセシビリティツリーから除外します。                                      |
+|                | `tabIndex`           | キーボードフォーカスを有効にしたい場合は`0`、無効にしたい場合は`-1`を設定します。 |
```
Comment (hirotomoyamada) [inline docs/contents/components/media-and-icons/icon/accessibility.ja.mdx:15]: Please remove line.

File: docs/contents/components/media-and-icons/icon/accessibility.ja.mdx [hunk 2]
```diff
@@ -0,0 +1,15 @@
+---
+title: Icon
+description: "`Icon`は、プロジェクトに使用できる一般的なアイコンコンポーネントです。"
+package_name: "@yamada-ui/react"
+order: 3
+with_description: true
+tab: アクセシビリティ
+---
+
+## ARIAロールと属性
```
Comment (hirotomoyamada) [inline docs/contents/components/media-and-icons/icon/accessibility.ja.mdx:10]: Add `role="img"` row.

File: docs/contents/components/media-and-icons/icon/accessibility.mdx [hunk 3]
```diff
@@ -0,0 +1,15 @@
+---
+title: Icon
+description: "`Icon` is a general icon component that can be used in your projects."
+package_name: "@yamada-ui/react"
+order: 3
+with_description: true
+tab: Accessibility
+---
+
+## ARIA Roles and Attributes
```
Comment (hirotomoyamada) [inline docs/contents/components/media-and-icons/icon/accessibility.mdx:10]: Add `role="img"` row.

File: docs/contents/components/media-and-icons/icon/accessibility.mdx [hunk 4]
```diff
@@ -0,0 +1,15 @@
...
+
+| Component | Roles and Attributes | Usage                                                          |
+| --------- | -------------------- | -------------------------------------------------------------- |
+| `Icon`    | `aria-hidden="true"` | Excludes the element from the accessibility tree.              |
+|           | `tabIndex`           | Set it to `0` to enable keyboard focus, or `-1` to disable it. |
```
Comment (hirotomoyamada) [inline docs/contents/components/media-and-icons/icon/accessibility.mdx:15]: Please remove line.

File: docs/contents/components/media-and-icons/icon/accessibility.ja.mdx (hunk 1 above)
Comment (kurakke) [reply docs/contents/components/media-and-icons/icon/accessibility.ja.mdx]: fixed

File: docs/contents/components/media-and-icons/icon/accessibility.ja.mdx (hunk 2 above)
Comment (kurakke) [reply docs/contents/components/media-and-icons/icon/accessibility.ja.mdx:10]: fixed

**Review by hirotomoyamada — APPROVED** [review]

**Pull request merged**